src/LSPEncoder.h

```diff
@@ -25,8 +25,12 @@ static	const	int		PAULA_REPLAY_RATE_MAX = 28800;
 static const int		LSP_INSTRUMENT_MAX = 2600;		// theoretical max is 32767/12
 static const int		LSP_CMDWORD_MAX = 255 * 3;
 
+#ifndef MACOS_LINUX
```

Comment:
I don't get this ifndef MACOS_LINUX, does it mean there is no LSP_VERSION defined on MAC?

Reply:
Actually this `#ifndef` is superfluous.  I added in the `#ifndef LSP_MAJOR_VERSION` on next line whilst making sure that Windows version is building both by using the .sln or with CMake at the command line.

Reply:
I got rid of the unneeded #ifndef